.travis.yml

```diff
@@ -13,3 +13,14 @@ sudo: false
 cache:
   directories:
     - $HOME/.yarn-cache
+deploy:
+  provider: releases
+  api_key:
+    secure: YeKIPlHkpDep5GB/VXzs2XyF2ym5Lh2nQ9XMsSntvxJ4K/MR0xRpwTxFN6TH1o4CW9RvlBGhhrWdeVGyMqEwnVY9OUx7jPkfKSkKBZZYx5DFMCuXxTJ0AgxT7IQjrsuweDdOWc5wdyKyxeJEJRnPfp4GOSPd8pJYSNKpJdmSzgM=
```

Comment:
Is this something that can be public? Wouldn't it allow anyone to access our github account?